GitUI/CommandsDialogs/BrowseDialog/DashboardControl/DashboardItem.Designer.cs

```diff
@@ -120,7 +126,7 @@ private void InitializeComponent()
             // 
             // DashboardItem
             // 
-            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Inherit;
```

Comment:
Controls should be with AutoScaleMode.Inherit, because we change form `AutoScaleMode` depending on app settings